releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,9 +4,15 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.11 - March, 24 2020
+* Updated [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
```

Comment:
Do we include documentation updates as a part of release notes? Typically, if something requires an update to the version of the extension, then we consider it to be a change in that release and include it in release notes. Is that the case here?